node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const filename = process.argv[2];
+
+if (filename === 'cunningham.txt' || filename === 'dijkstra.txt' || filename === 'hipster-ipsum.txt' || filename === 'hopper.txt') {
+  fs.readFile(filename, 'utf8', callBack);
+} else {
+  console.log('INVALID');
+}
```

Comment:
To be clear, if new `*.txt` files are added to the directory, `read-any.js` should be able to read those without modifying the source code. This isn't a validation issue, it's a "reads any text file the user specifies" issue.

---

![image](https://user-images.githubusercontent.com/16143771/159053631-f4306e9a-add8-42b0-91a4-b5dda4442c05.png)